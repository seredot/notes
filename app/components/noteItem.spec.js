import "react-native";
import React from "react";
import NoteItem from "./noteItem";

import renderer from "react-test-renderer";

describe("Testing NoteItem component", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<NoteItem />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
