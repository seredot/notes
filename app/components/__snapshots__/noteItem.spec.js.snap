// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NoteItem component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#dbddde",
        "overflow": "hidden",
      },
      Object {},
      Array [
        Object {
          "backgroundColor": "#F2EB61",
        },
      ],
    ]
  }
>
  <View
    onLayout={[Function]}
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        Object {},
        Object {
          "left": 0,
        },
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
    >
      <View
        style={
          Object {
            "backgroundColor": "#F2EB61",
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "#fff",
              "borderColor": "#ddd",
              "borderWidth": 1,
              "color": "#333",
              "fontSize": 16,
              "marginHorizontal": 8,
              "marginVertical": 5,
              "paddingHorizontal": 9,
              "paddingVertical": 8,
            }
          }
        />
      </View>
    </View>
  </View>
  <View />
  <View />
</View>
`;
